projects/canopy/src/lib/button/button.notes.ts

```diff
@@ -55,9 +70,42 @@ You can create an **icon only** button by setting \`iconButton\` to \`true\`. Yo
 | \`\`fullWidth\`\` | If the button has to span full width or not. For 'sm' and 'md' sized screens, the button will always be full width and this input has no affect | boolean | false | No |
 | \`\`disabled\`\` | Programmatically disable the button via this property | boolean | false | No |
 | \`\`loading\`\` | If the button shows a loading spinner and is also disabled | boolean | false | No |
-| \`\`iconPosition\`\` | The position of the icon in the button | string | right | No |
+| \`\`iconPosition\`\` | The position of a single icon in the button | string | right | No |
 | \`\`iconButton\`\` | The button displays an icon only | boolean | false | No |
 
+### Button toggle directive
+
+A button can be used specifically as a toggle. In this case the applied styles are slightly different than a regular button and you **must** use a \`button\` tag.
```

Comment:
```suggestion
A button can be used specifically as a toggle, for example when opening and closing a filter panel. In this case the applied styles are slightly different than a regular button and you **must** use a \`button\` tag for accessibility reasons - if you add it to a link or any other element, it will throw an error. 
```